### PR TITLE
Ui dev keith

### PR DESCRIPTION
**Pull request by KeithCissell**
Code for building a canvas containing a grid of cells to be used as visual representation of the GOL simulation.

### CODE REVIEW

File: GameOfLifeViewController.py [hunk 1]
```diff
@@ -0,0 +1 @@
+import GameOfLifeViewContorller
```
Comment (enki211) [inline GameOfLifeViewController.py]: This file does not seem to be needed

File: DisplayGrid.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            self.canvas.itemconfig(item_id, fill="black")
+        self.after(delay, lambda: self.redraw(delay))
+
+if __name__ == "__main__":
+    gridSize = 30   # row-by-column dimention of the grid
+    
+    root = Tk()
+    app = App(gridSize, master=root)
+    app.mainloop()
+    root.destroy()
```
Comment (enki211) [inline DisplayGrid.py]: newline at EOF

File: GameOfLifeViewController.py (hunk 1 above)
Comment (dwilliams224) [inline GameOfLifeViewController.py]: @enki211 This file will be used to manage receive updates from the Game of Life files, update the gridView, and display button controls for the app. Its basically the piece that will bring the app together in the end.

Comment (enki211) [inline GameOfLifeViewController.py]: @dwilliams224 when that happens then this file will be needed. At the moment it does not seem to be needed

**Pull request merged**